docs/sdks/stash/uid2-sdk-ref-mobile-gma.md

```diff
@@ -0,0 +1,99 @@
+# Developer Guide for UID2 GMA Plugin (iOS and Android)
+
+As of 2023-07-15 plugin version:
+* iOS V 0.2.0
+* Android V 0.4.0
+
+## Introduction
+
+The UID2 GMA Plugin is pre-release software and should be used as such.
+
+The UID2 GMA Plugin enables apps that use Google Mobile Ad (GMA) SDK to
+include UID2 data in their ad requests. This is done automatically, with
+no direct coding needed from the app developer, by:
+
+-   The Google Mobile Ad SDK
+
+-   The UID2 SDK
+
+-   The UID2 GMA Plugin
+
+## iOS 
+
+Refer to the following sections for iOS:
+
+-   Requirements
+
+-   Installation
```

Comment:
don't think we need line 23-27 - it's redundant

Reply:
Noted for the GMA doc + IMA later... thx!